OpenEdXMobile/src/main/java/org/edx/mobile/event/AppUpdatedEvent.java

```diff
@@ -0,0 +1,22 @@
+package org.edx.mobile.event;
+
+import android.support.annotation.NonNull;
+
+public class AppUpdatedEvent {
+    @NonNull
+    public final long previousVersionCode;
+    @NonNull
+    public final long currVersionCode;
+    @NonNull
+    public final String previousVersionName;
```

Comment:
This variable will be `null` the first time app upgrades after this branch merges into master, cuz we wont have any previous version name stored in our preferences and our `getString` function returns `null` in such a case.